### DISCUSSION TITLE
Update NESDIS weekly greenness fraction

### DISCUSSION THREAD

**Discussion by nchawang** [NASA-LIS/LISF#948] [Q&A]
I was trying to update the greenness fraction in LIS for the period of 2008-2010 over the Indian domain, so i used the NESDIS weekly option. When i run LIS, pointing to the directory where NESDIS weekly greenness fraction is present nothing happens, no output is getting generated, at the same time no error is shown and the lislog file is empty.

the gvf was downloaded from : https://www.star.nesdis.noaa.gov/smcd/emb/vci/gvps/gvps_realtime_data.php
(GRIB directed me to : https://www.star.nesdis.noaa.gov/pub/corp/scsb/wguo/data/GRIB/GVF_v4.0/Final/ where i downloaded the D08-D10 files)

I've attached the lislog and lisconfig files. Have i missed something? do i have to make changes in ldt.config or was the gvf files wrong?

EDIT : The issue is resolved, the problem was that the fortran code looks for NPR.VGWG.D* whereas the files were in NPR.VGWG.NN.D* from nesdis (same as the link above). 
The code lies in lis/params/gfrac/NESDISweekly/read_NESDISgfrac.F90 for anyone in the future.

Thanks

**Comment by bmcandr** [comment]
Hi @nchawang,

Is any error or traceback printed to the terminal or some other file (e.g., *.err)? It looks like LIS is stopping well before reading the greenness fraction files.

**Reply by nchawang** [reply]
Hi @bmcandr,
it shows no error on the terminal, but nothing is being shown as well when I tail the lislog file, if I use the same and change the greenness parameter to LDT, it runs normally, changing it to NESDIS weekly, it just remains stuck.

If I want to use the NESDIS weekly for the same domain do I have to make any changes in the ldt.config file could the issue be in the lis_input.nc file?

**Reply by nchawang** [reply]
Hi @bmcandr ,
I have changed the ldt.config file and generated a lis_input.nc file without greenness fraction. I tried implementing the option "NESDIS weekly" the same occurred, no error is shown in the terminal but lislog file also shows nothing and no output, only thing left is to stop the run. 
I have changed the directory to a random directory to check, surprisingly it shows no error of a wrong directory the same thing happens in which the run is stuck in this.
To just check, I put LDT as greenness input using this input which does not have a generated greenness fraction, it shows, an error at "read_gfracclimo".

Could there be a requirement for changes in the input text files? (forcings_variables.txt, etc)
